application/features/analytics.feature

```diff
@@ -3,9 +3,11 @@ Feature: User
   In order to report about mfa usage
   I need to be able to count active users with diffent kinds of verified mfa records
 
+  Background:
+    Given the user store is empty
```

Comment:
This is somewhat subjective, but it seems to be more helpful in general to keep Behat statements as customer-friendly as reasonably possible. For example, rather than saying "the user store is empty" it might be better to say "there are no users" (or possibly "there are no users in the database").

The thought behind it is that the *.feature files should contain language that the customer / product owner can read and understand to bring consensus on what qualifies as "done". Since this are tests for components without our own system, that customer interaction is even less likely to happen, but keeping that goal still seems valuable.

Reply:
OK, thanks. I'll write feature statements with that in mind.